inc/miniRT.h

```diff
@@ -21,7 +21,7 @@
 # include "libft.h"
 # include "mlx.h"
 
-// Macros for Type Objects
+// Macros for Objects Types
```

Comment:
sans le s pour object
```suggestion
// Macros for Object Types
```